tests/ui/impl-trait/equality-in-canonical-query.clone.stderr

```diff
@@ -19,5 +19,7 @@ LL |     same_output(foo, rpit);
 LL |     same_output(foo, rpit);
    |     ^^^^^^^^^^^^^^^^^^^^^^
 
+note: if you are using a nightly compiler, please make sure that you have updated to the latest nightly
```

Comment:
Is there any point in having "if you are using a nightly compiler"? Why not check `tcx.sess.is_nightly_build()` directly?

Reply:
I thought install_ice_hook and report_ice were so early in rustc_driver_impl that we didn't have a tcx available?

Reply:
I believe you can use `rustc_feature::UnstableFeatures::from_environment(None).is_nightly_build();` for this instead.